docs/content/posts/2023-11-13-release-1.17.2.md

```diff
@@ -0,0 +1,204 @@
+  <version>1.17.2</version>
+</dependency>
+```
+ 
+## Binaries
+ 
+You can find the binaries on the updated [Downloads page]({{< relref "downloads" >}}).
+ 
+## Docker Images
+ 
+* [library/flink](https://hub.docker.com/_/flink/tags?page=1&name=1.17.2) (official images)
+* [apache/flink](https://hub.docker.com/r/apache/flink/tags?page=1&name=1.17.2) (ASF repository)
+ 
+## PyPi
+ 
+* [apache-flink==1.17.2](https://pypi.org/project/apache-flink/1.17.2/)
+
+# Release Notes
+
+        Release Notes - Flink - Version 1.17.2
+       
+<h2>        Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27415'>FLINK-27415</a>] -         Read empty csv file throws exception in FileSystem table connector
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28513'>FLINK-28513</a>] -         Flink Table API CSV streaming sink throws SerializedThrowable exception
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29913'>FLINK-29913</a>] -         Shared state would be discarded by mistake when maxConcurrentCheckpoint&gt;1
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-30559'>FLINK-30559</a>] -         May get wrong result for `if` expression if it&#39;s string data type
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-30596'>FLINK-30596</a>] -         Multiple POST /jars/:jarid/run requests with the same jobId, runs duplicate jobs
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-30751'>FLINK-30751</a>] -         Remove references to disableDataSync in RocksDB documentation
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-30966'>FLINK-30966</a>] -         Flink SQL IF FUNCTION logic error
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-31139'>FLINK-31139</a>] -         not upload empty state changelog file
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-31519'>FLINK-31519</a>] -         The watermark alignment docs is outdated after FLIP-217 finished
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-31812'>FLINK-31812</a>] -         SavePoint from /jars/:jarid:/run api on body is not anymore set to null if empty
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-31967'>FLINK-31967</a>] -         SQL with LAG function NullPointerException
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-31974'>FLINK-31974</a>] -         JobManager crashes after KubernetesClientException exception with FatalExitExceptionHandler
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32023'>FLINK-32023</a>] -         execution.buffer-timeout cannot be set to -1 ms
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32034'>FLINK-32034</a>] -         Python&#39;s DistUtils is deprecated as of 3.10
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32049'>FLINK-32049</a>] -         CoordinatedSourceRescaleITCase.testDownscaling fails on AZP
```

Comment:
This one is a test stability fix and should be removed from here

Reply:
Thanks for the catch, resolved.